easel/lib/screens/creator_hub/creator_hub_screen.dart

```diff
@@ -389,8 +353,8 @@ class NFTsViewBuilder extends StatelessWidget {
 class BuildGridView extends StatelessWidget {
   final List<NFT> nftsList;
   final WidgetBuilder onEmptyList;
-
-  const BuildGridView({Key? key, required this.nftsList, required this.onEmptyList}) : super(key: key);
+  final CollectionType collectionType;
```

Comment:
Whats the reason for add this as a part of constructor. We can get it through provider in the NFTGridViewItem without passing,

Reply:
Ok, I am updating it now. Thank you

Reply:
@Kamran-Khan-Gene  whats the update on this?

Reply:
I'ts done.